### DISCUSSION TITLE
Dynamic classes show type error

### DISCUSSION THREAD

**Discussion by rohit8** [muhammadsammy/tailwindcss-classnames#384] [Q&A] [answered]
I'm using `tailwindcss-classnames@3.0.2` and I'm seeing a type error with Dynamic classes:

<img width="416" alt="Screen Shot 2022-02-28 at 1 52 49 AM" src="https://user-images.githubusercontent.com/2968479/155937968-f53b3ff0-44b0-4d65-8502-fb5a71549660.png">

The type error is:
```
Argument of type '{ TAILWIND_STRING: boolean; }' is not assignable to parameter of type '"TAILWIND_STRING"'.
```

How can I uses dynamic classes?

**Comment by bstro** [comment, marked as answer]
Hey there, you're close! What you're looking for is this:
```
import classnames, {TW} from 'tailwindcss-classnames'

const styles = classnames(
  TW.textColor({
    "text-gray-600": !isActive
  }),
  TW.display("flex"),
  …etc
)
```

Good luck!

**Reply by karltaylor** [reply]
This would be useful in the `README.md` as the current documentation uses the old style which will throw errors

**Reply by MaxJLerman** [reply]
I've been trying to figure out how to fix this error for two days using the docs but no one else seems to know this is a problem as of yet. This is the first occurence of someone else having the issue I'm having so thank you so much for pointing it out. @bstro you are a legend
